iphone/Maps/UI/Settings/MWMTTSSettingsViewController.mm

```diff
@@ -17,6 +17,7 @@
 enum class Section
 {
   VoiceInstructions,
+  StreetNames,
```

Comment:
This is not serialized anywhere, right?

Reply:
@dvdmrtnz can you help me with these iPhone questions?